### DISCUSSION TITLE
Migration to add enum column not working

### DISCUSSION THREAD

**Discussion by KyleAMathews** [electric-sql/electric#1105] [Feature requests]
Which is odd as this would work if I created the enum when creating the table.

```sql
CREATE TYPE application_status AS ENUM ('applied', 'rejected');
ALTER TABLE read_jobs ADD COLUMN why application_status;
```

I'm assuming this is a bug?

**Comment by linear** [comment]
<p><a href="https://linear.app/electric-sql/issue/VAX-1754/migration-to-add-enum-column-not-working">VAX-1754 Migration to add enum column not working</a></p>

**Comment by samwillis** [comment]
Hey 👋 

Adding enum columns to an already electrified table hasn't been implemented yet. There is a little complexity we need to solve with the ordering of operations that doesn't apply when electrifying an existing table with them. It's on the to do list.

**Comment by msfstef** [comment]
This is a current limitation with ENUM types, see [warning at the bottom of the types docs](https://electric-sql.com/docs/usage/data-modelling/types).

Since this is a known and documented limitation that might take a bit of time to resolve I will convert this issue into a discussion.
